recipe/meta.yaml

```
@@ -26,37 +26,39 @@ requirements:
   build:
     # pythran code needs clang-cl on windows
     - clang                 # [win]
+    - lld     # [win]
     - {{ compiler("c") }}
     - {{ compiler("cxx") }}
     - llvm-openmp >=4.0.1   # [osx]
     - m2-patch              # [win]
     - patch                 # [not win]
   host:
     - python
-    - cython >=0.29.24,<3.0
+    - cython >=0.29.24
     - llvm-openmp >=4.0.01  # [osx]
-    - numpy   1.19  # [py<310]
-    - numpy   1.21  # [py==310]
-    - numpy   1.23  # [py>=311]
+    - numpy {{ numpy }}
     - packaging
     - pip
     - pythran
-    - setuptools <=59.4
+    - setuptools
     - wheel
+    - meson-python 0.12.1
+    - lazy_loader >=0.1
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - imageio >=2.16.2
+    - imageio >=2.4.1
+    - lazy_loader >=0.1.0
```

```suggestion
    - lazy_loader >=0.1
```

https://github.com/scikit-image/scikit-image/blob/v0.20.0/pyproject.toml#L48

---

Thanks Charles! Comitted.